.gitmodules

```diff
@@ -34,7 +34,7 @@
     ignore = dirty
     path = third_party/gloo
     url = https://github.com/ROCmSoftwarePlatform/gloo
-    branch = release/2.0
+    branch = rocm6.0_internal_testing
```

Comment:
For gloo, can we use the release/2.0 and update this with the fix.

Reply:
@pruthvistony Updated gloo with release/2.0 branch plus the fix. Performed PyTorch build/test and uploaded logs.